lib/mayaUsd/resources/scripts/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@ list(APPEND scripts_src
 )
 
 install(FILES ${scripts_src}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/scripts
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/scripts
```

Comment:
The installer knew nothing abou this top-level "scripts" folder. It needs to be under "lib" where the other lib stuff is.